crates/matrix-sdk-common/src/linked_chunk/builder.rs

```diff
@@ -480,7 +583,7 @@ pub enum LinkedChunkBuilderTestError {
 }
 
 #[cfg(test)]
-mod tests {
+mod linked_builder_test_tests {
```

Comment:
uh? we do use `tests` by convention everywhere in the code base, can we maintain this?

Reply:
Another `mod tests` exists in this file. I can merge both of them if you prefer.